tests/ui/type_repetition_in_bounds.stderr

```diff
@@ -0,0 +1,14 @@
+error: This type has already been used as a bound predicate
```

Comment:
NIT: error message start lowercase

It also would be great if you could use the `span_help_and_lint` function and add the message:
```
help: consider combining the bounds: `T: Copy + Clone`
```

Reply:
Will do, is `T: Copy + Clone` just an example to be put verbatim or should I replace it with the actual bounds and type parameters used in the offending code?

Reply:
It was just an example, see review below 😉